### PR TITLE
PR09 batch 4

### PR DESCRIPTION
**Pull request by HughKelley**
chunk of #28602

### CODE REVIEW

**Comment by HughKelley** [comment]
I'm unsure how to format line 133 in pandas/io/excel/_base.py. flake8 is picking up the space argument required to correctly format the list being inserted as an inappropriate indent.

**Comment by WillAyd** [comment]
> I'm unsure how to format line 133 in pandas/io/excel/_base.py. flake8 is picking up the space argument required to correctly format the list being inserted as an inappropriate indent.

What was the issue you faced previously?

**Comment by HughKelley** [comment]
> What was the issue you faced previously?

Was adding indentation to the docstring while adding a period to a parameter description here: https://github.com/pandas-dev/pandas/pull/29466/commits/020e253c7f6eff26fa02877628e9ee4c5ec65e1a

it required changing the indentation argument from 4 spaces to 8 to match the rest of the string.

**Comment by WillAyd** [comment]
Hmm can you post the built HTML doc? Looks OK on stable docs wouldn't expect to have to change the indent

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_excel.html

**Comment by HughKelley** [comment]
I was adding a period and thought the indentation should match throughout the string like below but if it's ok I'll undo that change.

``` python
example = (
    """
summary

Parameters
----------
parameter_1 :  str
    Description.
"""
)
``` 
changed to 
```
example = (
    """
    summary

    Parameters
    ----------
    parameter_1 :  str
        Description.
    """
   )

**Comment by HughKelley** [comment]
@WillAyd `./scripts/validate_docstrings.py` wasn't picking up the period at the end of the `use_cols` parameter description. 

Adding an indent to the doc_string as I did in 020e253 resolved that but required adding 4 more spaces to the separator argument in `fill()` which created the `E131 hanging indent` error

**Comment by WillAyd** [comment]
Are we talking about `usecols` or `na_values`? I think I'm confused as to the source.

In any case I think OK to tweak slightly to get it to pass validation (i.e. adding text after bullets, or something to the effect)

@datapythonista

**Comment by HughKelley** [comment]
> Are we talking about `usecols` or `na_values`? I think I'm confused as to the source.

The original error i was trying to fix was the a missing period for `use_cols`. I fixed that by indenting the string. that created a new error in `na_values` where in order to format the list correctly I had to add spaces to the separator argument which flake8 was then picking up as a hanging indent.

File: pandas/core/tools/datetimes.py [hunk 1]
```diff
@@ -577,14 +577,13 @@ def to_datetime(
...
-           or DataFrame/dict-like
-
+    arg : int, float, str, datetime, list, tuple, 1-d array, Series DataFrame/dict-like
+        The object to convert to a datetime.
     errors : {'ignore', 'raise', 'coerce'}, default 'raise'
-
-        - If 'raise', then invalid parsing will raise an exception
-        - If 'coerce', then invalid parsing will be set as NaT
-        - If 'ignore', then invalid parsing will return the input
+        Behaves as:
```
Comment (WillAyd) [inline pandas/core/tools/datetimes.py]: Was this required to get validation to work?

Comment (HughKelley) [reply pandas/core/tools/datetimes.py]: Yes, otherwise it raises error PR08, that the description should start with a capital letter.

**Review by WillAyd — APPROVED** [review]
lgtm @datapythonista if you care to take a look

File: pandas/core/reshape/merge.py [hunk 2]
```diff
@@ -211,16 +212,17 @@ def merge_ordered(
...
-        * inner: use intersection of keys from both frames (SQL: inner join)
+        * inner: use intersection of keys from both frames (SQL: inner join).
 
     Returns
     -------
-    merged : DataFrame
-        The output type will the be same as 'left', if it is a subclass
-        of DataFrame.
+    DataFrame
+        The merged DataFrame The output type will the be same as
```
Comment (datapythonista) [inline pandas/core/reshape/merge.py]:
```suggestion
        The merged DataFrame output type will the be same as
```

File: pandas/core/tools/datetimes.py (hunk 1 above)
Comment (datapythonista) [reply pandas/core/tools/datetimes.py]: You can remove all this. The new version of the script doesn't fail if the description starts with a list. And we'll move to the new version of the script soon.

**Review by datapythonista — COMMENTED** [review]
Thanks @HughKelley, really nice PR.

Just a typo, and some additions that are not necessary, other than that looks perfect.

**Review by datapythonista — APPROVED** [review]
If you want to remove the remaining `Behaves as:` and `Behavior:` that would be great, but lgtm

**Comment by HughKelley** [comment]
> If you want to remove the remaining `Behaves as:` and `Behavior:` that would be great, but lgtm

ah thanks

**Comment by HughKelley** [comment]
@WillAyd and @datapythonista checks are green. Thanks very much for the help on this one!

**Comment by datapythonista** [comment]
Thanks a lot for helping fix the documentation @HughKelley

**Pull request merged**